src/snowflake/snowpark/modin/plugin/extensions/base_overrides.py

```diff
@@ -183,6 +182,22 @@ def filter(
     pass  # pragma: no cover
 
 
+@register_base_not_implemented()
+def interpolate(
+    self,
+    method="linear",
+    *,
+    axis=0,
```

Comment:
where does this come from? i though we are only removing dataframe in this pr, why do we have a base override ?

Reply:
In modin 0.28.1 this method is defined on the base class, but our vendored copy still had it defined on DataFrame.